cip-00XX.md

```diff
@@ -0,0 +1,73 @@
+    CIP: ??
+    Title: Updated AT fees
+    Author: frank_the_tank, jjos, burstjack
+    Comments-Summary: No comments yet.
+    Comments-URI: 
+    Status: Draft
+    Type: Hard Fork
+    Created: 2019-04-07
+
+
+## Motivation
+
+With Burst, smart contracts are implemented using Automated Transactions (AT), a technology created by the CIYAM developers. Automated Transactions are turing-complete and thus have a potentially infinite number of use cases. Beside this possible infinite number of use cases the current fee structure for AT on Burst-chain is counterproductive.
+To attract business and developers for an investment/involvement with the ATs on the Burstcoin blockchain the fee structure needs to be reviewed and changed.
+
+
+## Abstract
+
+Any user with an BURST-Account is able to create an AT via the wallet or the corresponding API call. Unlike other blockchains, BURST does not have its own colored coin for running ATs.Registrating ATs on chain is supported by fees in BURST while execution of AT code is supported by the AT BURST balance.Initial setup of an AT costs BURST but the neck-breaker of the current implementation is the fee structure for keeping the AT alive. While in a recent hard-fork (CIP03) the new minimum fee of 0.00735 BURST was introduced (previous value was 1 BURST), the adaptation for the AT handling was forgotten or overlooked. Also the number of function calls an AT can do within a block was not adjusted which forces a complex AT to be dispensed over several blocks before the AT got finished.
+Thus, in this proposal a new fee scheme for AT creation and execution is suggested.
+
+
+## Specification
+
+The parameters for the AT fee structure are handled in the AT_Constants.java file of Burst Reference Software (BRS). We propose to update the parameters in the following way (constants not changed are not listed):
+
+
+| Constant | AT version1 | AT version 2|
+|----------|-------------|-------------|
+| STEP_FEE |  0.1 BURST  | FEE_QUANT/10|
+| MAX_STEPS| 2000        | 100000      |
+| COST_PER_PAGE| 1 BURST | 10*FEE_QUANT|
+
+### Explanation of Parameters
+
```

Comment:
MIN_FEE
Currently unused, this constant is not referenced in BRS code. However, there is already a fee for every possible AT actions.

Reply:
This will be removed separately then.